### PR TITLE
satellite/payments: credit card selection

### PR DESCRIPTION
**Pull request by crawter**
What: 
Payments service and API extended with new functionality
Why:

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

File: satellite/payments/stripecoinpayments/creditcards.go [hunk 1]
```diff
@@ -82,3 +82,26 @@ func (creditCards *creditCards) Add(ctx context.Context, userID uuid.UUID, cardT
...
+		Params:        stripe.Params{},
+		DefaultSource: stripe.String(string(cardID)),
+	}
+
+	_, err = creditCards.service.stripeClient.Customers.Update(customerID, params)
+	if err != nil {
+		return Error.Wrap(err)
+	}
+
+	return nil
```
Comment (egonelbre) [inline satellite/payments/stripecoinpayments/creditcards.go]: You can use `return Error.Wrap(err)` with nil errors. It `Error.Wrap(nil)` returns `nil`.

Comment (crawter) [reply satellite/payments/stripecoinpayments/creditcards.go]: thanks!

File: satellite/console/consoleweb/consoleapi/payments.go [hunk 2]
```diff
@@ -66,50 +73,131 @@ func (p *Payments) AccountBalance(w http.ResponseWriter, r *http.Request) {
...
 		Balance int64 `json:"balance"`
 	}
 
-	balanceResponse.Balance = balance
+	responseBody.Balance = balance
 
-	err = json.NewEncoder(w).Encode(balanceResponse)
+	err = json.NewEncoder(w).Encode(responseBody)
 	if err != nil {
 		p.log.Error("failed to write json balance response", zap.Error(err))
```
Comment (Qweder93) [inline satellite/console/consoleweb/consoleapi/payments.go]: you can add return here for consistency

File: satellite/console/consoleweb/consoleapi/payments.go [hunk 3]
```diff
@@ -66,50 +73,131 @@ func (p *Payments) AccountBalance(w http.ResponseWriter, r *http.Request) {
...
+
+	var responseBody struct {
+		Cards []payments.CreditCard `json:"cards"`
+	}
+
+	responseBody.Cards = cards
+
+	err = json.NewEncoder(w).Encode(responseBody)
+	if err != nil {
+		p.log.Error("failed to write json list cards response", zap.Error(err))
```
Comment (Qweder93) [inline satellite/console/consoleweb/consoleapi/payments.go]: same here

File: satellite/console/consoleweb/consoleapi/payments.go [hunk 4]
```diff
@@ -4,19 +4,23 @@
...
-	"context"
 	"encoding/json"
+	"io/ioutil"
 	"net/http"
-	"strings"
 
+	"github.com/gorilla/mux"
+	"github.com/zeebo/errs"
 	"go.uber.org/zap"
-	monkit "gopkg.in/spacemonkeygo/monkit.v2"
```
Comment (Qweder93) [inline satellite/console/consoleweb/consoleapi/payments.go]: keep monkit

**Pull request merged**